src/markdown-service.ts

```diff
@@ -16,20 +16,22 @@ export const getMarkdownTableCell = ({
   metricUnit,
   metricType
 }: MarkdownTableCellInterface): string => {
-  if (metricType === 'performance') {
-    return `[${currentValue}${metricUnit} ${isRegression ? '🔴' : '🟢'}](## "Performance has  ${
-      isRegression ? 'decreased in ' : 'improved in +'
-    }${diffValue} points")`
-  } else if (metricType === 'lcp' || metricType === 'tbt') {
-    return `[${currentValue} ms ${isRegression ? '🔴' : '🟢'}](## "The ${metricType} has ${
-      isRegression ? 'increased in +' : 'decreased in '
-    }${diffValue} ms")`
-  } else if (metricType === 'cls') {
-    return `[${currentValue} ${isRegression ? '🔴' : '🟢'}](## "The CLS has ${
-      isRegression ? 'increased in +' : 'decreased in'
-    } ${diffValue}")`
-  } else {
-    return ''
+  switch (metricType) {
```

Comment:
@andresz1 Here I refactored this to use a switch instead of a chain of if/else and I think this improves readability too.

Let me know what you think.